src/app/Dependencies.tsx

```diff
@@ -106,9 +107,15 @@ const DisplayDependencies: FC<ExtensionItemProps> = ({ dependencies }) => {
     setElements(getLayoutedElements(dagreGraph, initialElements));
   }, [dependencies]);
 
+  const style = {
+    height: `${height}px`,
```

Comment:
Pixels are not good as changes to the available screen height are not reflected. Please make the solution using CSS only.